components/History/History.tsx

```diff
@@ -96,22 +170,23 @@ const History: React.FunctionComponent<HistoryProps> = ({
       <Modal
         animationType="slide"
         transparent={false}
-        visible={isTxDetailModalShowing}
-        onRequestClose={() => setTxDetailModalShowing(false)}>
-        <TxDetail
-          index={transactionsSorted.findIndex((tx: TransactionType) => tx.txid === txDetail.txid)}
-          length={transactionsSorted.length}
-          tx={txDetail}
-          closeModal={() => setTxDetailModalShowing(false)}
-          openModal={() => setTxDetailModalShowing(true)}
+        visible={isValueTransferDetailModalShowing}
+        onRequestClose={() => setValueTransferDetailModalShowing(false)}>
+        <ValueTransferDetail
+          index={valueTransferDetailIndex}
+          length={valueTransfersSorted.length}
+          totalLength={valueTransfers.length}
```

Comment:
i don't understand how the sorted length could be different to the value transfers length?

Reply:
in the fetch function you can see the `slice` with the first 50 lines... means the sorted list can have 50 and the total list can have N > 50...